### PR TITLE
Refactoring forms in complete-invitation

### PR DESCRIPTION
**Pull request by Andresbl123**
The AidBudgetsForm, the branchesForm, the EventsForm, the PayrollsForm, and the ProjectsForm are the same, they use the AssignmentForm but define different titles, and so on.

Refactor the CompleteInvitation and the AssignmentForm components to delete all of the other forms or create a new generic componentForm if necessary (I think it is not).

### CODE REVIEW

**Review by LinconOssa — APPROVED** [review]
It meets the requirements and would be to modify the storybook as discussed to make the new change.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/users/complete-invitation/index.tsx [hunk 1]
```diff
@@ -8,12 +8,8 @@ import { IVerificationData } from "@pages/privileges/outlets/users/complete-invi
 import { EMessageType } from "@src/types/messages.types";
 import { useState } from "react";
 import { useNavigate, useParams } from "react-router-dom";
-import { AidBudgetsForm } from "../edit-user/forms/AidBudgetsForm";
-import { BranchesForm } from "../edit-user/forms/BranchesForm";
-import { EventsForm } from "../edit-user/forms/EventsForm";
+import { InitialiserForm } from "../edit-user/forms/InitialiserForm";
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/pages/privileges/outlets/users/complete-invitation/index.tsx:11]: resuelto

File: src/pages/privileges/outlets/users/edit-user/forms/InitialiserForm/InitialiserForm.stories.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 import { BrowserRouter } from "react-router-dom";
-import { AidBudgetsForm, AidBudgetsFormProps } from "./index";
+import { InitialiserForm, IInitialiserForm } from "./index";
 import { aidBudgetsFormEditUser } from "@mocks/apps/privileges/users/aidBudgetsForm.mock";
 import { IAssignmentFormEntry } from "../../../types/forms.types";
 import { StoryFn } from "@storybook/react";
 
 const story = {
-  components: [AidBudgetsForm],
+  components: [InitialiserForm],
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/pages/privileges/outlets/users/edit-user/forms/InitialiserForm/InitialiserForm.stories.tsx:9]: resuelto

File: src/pages/privileges/outlets/users/edit-user/forms/InitialiserForm/index.tsx [hunk 3]
```diff
@@ -8,30 +8,29 @@ import { EMessageType } from "@src/types/messages.types";
...
-
-function AidBudgetsForm(props: AidBudgetsFormProps) {
+function InitialiserForm(props: IInitialiserForm) {
   const {
-    currentAidBudgetUnits,
+    dataOptionsForms,
     handleSubmit,
     withSubmitButtons,
     onHasChanges,
     readOnly,
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/pages/privileges/outlets/users/edit-user/forms/InitialiserForm/index.tsx:24]: resuelto

File: src/pages/privileges/outlets/users/edit-user/forms/InitialiserForm/index.tsx [hunk 4]
```diff
@@ -8,30 +8,29 @@ import { EMessageType } from "@src/types/messages.types";
...
   } = props;
-  const [aidBudgetUnits, setAidBudgetUnits] = useState(currentAidBudgetUnits);
+  const [aidBudgetUnits, setAidBudgetUnits] = useState(dataOptionsForms);
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/pages/privileges/outlets/users/edit-user/forms/InitialiserForm/index.tsx:26]: resuelto

File: src/pages/privileges/outlets/users/edit-user/forms/InitializerForm/InitializerForm.stories.tsx [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
 import { StoryFn } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+
+import { aidBudgetsFormEditUser } from "@mocks/apps/privileges/users/aidBudgetsForm.mock";
+import { IAssignmentFormEntry } from "@pages/privileges/outlets/users/types/forms.types";
+import { InitializerForm, IInitializerForm } from "./index";
 
 const story = {
-  components: [AidBudgetsForm],
+  components: [InitializerForm],
```
Comment (carin91) [inline src/pages/privileges/outlets/users/edit-user/forms/InitializerForm/InitializerForm.stories.tsx:10]: This form will be used more than 10 times on our page. The storybook is important for this component. Therefore, complete the storybook considering types, controls, and so on.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/users/edit-user/forms/InitializerForm/index.tsx [hunk 6]
```diff
@@ -8,33 +8,32 @@ import { EMessageType } from "@src/types/messages.types";
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/pages/privileges/outlets/users/edit-user/forms/InitializerForm/index.tsx:8]: resuelto

File: src/pages/privileges/outlets/users/edit-user/forms/InitializerForm/InitializerForm.stories.tsx (hunk 5 above)
Comment (Andresbl123) [reply src/pages/privileges/outlets/users/edit-user/forms/InitializerForm/InitializerForm.stories.tsx:10]: resuelto

**Review by carin91 — APPROVED** [review]

**Pull request merged**